app/assets/javascripts/views/business_support/areas_relator.js

```diff
@@ -1,52 +1,50 @@
 $(document).ready(function() {
   "use strict";
 
-  var $relatedAreasTextArea = $("#edition_areas"),
-      bindAreasSelection = function (selector, filter) {
-        $(selector).change(function () {
-          var self = this;
-          $('.areas-chkbx').each(function (i,e) {
-            if (e !== self) $(e).prop('checked', false);
-          });
-          $relatedAreasTextArea.select2("data", []);
-          if ($(this).is(':checked')) {
-            $relatedAreasTextArea.select2("data", $.map(areas, filter));
-          }
-        });
+  var $relatedAreasSelect = $("#edition_areas"),
+      bindAreasSelection,
+      areas;
+
+  areas = function () {
+    return $.map($relatedAreasSelect.find("option"), function (areaOption) {
+      var $areaOption = $(areaOption);
+
+      return {
+        id: $areaOption.val(),
+        text: $areaOption.text(),
+        country: $areaOption.data("country"),
+        type: $areaOption.data("type"),
       };
+    });
+  };
+
+  bindAreasSelection = function (selector, filter) {
+    $(selector).change(function () {
+      var self = this;
+      $('.areas-chkbx').each(function (i,e) {
+        if (e !== self) $(e).prop('checked', false);
+      });
+      $relatedAreasSelect.select2("data", []);
+      if ($(this).is(':checked')) {
+        $relatedAreasSelect.select2("data", $.map(areas(), filter));
+      }
+    });
+  };
 
   // http://ivaynberg.github.io/select2/select-2.1.html
-  $relatedAreasTextArea.select2({
+  $relatedAreasSelect.select2({
     width: "75%",
-    multiple: true,
```

Comment:
Yes! Actually, select2 complained that it would rather infer this from the <select> element's attributes. I tried to describe this in the commit message (with "- Remove a lot of custom select2 configuration"), but I can make that a bit clearer still if you'd like?